### PR TITLE
Configurable favicons

### LINKED ISSUE #4948: Configurable logo url and favicon
It would be nice to have a configurable logo in the config file like you did for the name of pootle site (`POOTLE_TITLE`) instead of creating a custom template or editing the css of the default one.

### PR DESCRIPTION
**Pull request by phlax**
this PR allows favicons to be customized for each pootle site.

~~it doesnt catch all possible favicons (eg android), but will allow configuration for common desktop browsers.~~

fixes #4948 

~~(still needs some changes to docs before landing)~~

### CODE REVIEW

File: docs/server/settings.rst [hunk 1]
```diff
@@ -182,6 +182,16 @@ Site-specific settings.
...
 
 
+.. setting:: POOTLE_CUSTOM_LOGO
+
+``POOTLE_FAVICONS_PATH``
+  Default: ``"/assets/favicon"``
+
+  .. versionadded:: 2.8
+
+  Customisable favicon path. Should not end with trailing slash.
```
Comment (unho) [inline docs/server/settings.rst]: Maybe we should add a Django check to ensure it ends with trailing slash.

File: docs/server/settings.rst [hunk 2]
```diff
@@ -182,6 +182,16 @@ Site-specific settings.
   Custom logo URL - this can be an absolute or relative URL.
 
 
+.. setting:: POOTLE_CUSTOM_LOGO
```
Comment (unho) [inline docs/server/settings.rst]: Wrong setting name.

File: docs/server/settings.rst (hunk 1 above)
Comment (phlax) [inline docs/server/settings.rst]: feels a bit like overkill, but we can if its an issue

File: docs/server/settings.rst (hunk 2 above)
Comment (phlax) [inline docs/server/settings.rst]: ill update...

**Comment by unho** [comment]
Also might be a good idea to include this new settings list in the release notes.

File: docs/server/settings.rst (hunk 1 above)
Comment (unho) [inline docs/server/settings.rst]: wfm

**Pull request merged**